### DISCUSSION TITLE
Login screen need wait long time about 30+ secs when boot after version NootedRed-ci1052-20231021-d8d7cd1

### DISCUSSION THREAD

**Discussion by laris** [ChefKissInc/NootedRed#222] [Q&A] [answered]
**Describe the bug**

**To Reproduce**
**Each time boot into login screen, the screen will be black background, but mouse can be show and move**

**Expected behavior**
**After boot to login screen, show login page not black screen**
**Screenshots**

**System Information:**
 **- macOS Version: [Sonoma v14.0]
 - CPU: [AMD Ryzen 7 5800U]**

**Additional context**
**Latest 3 version tested**

1. NootedRed-ci1052-20231021-d8d7cd1 (ci + number) (build date) (commit id) have issue
2. NootedRed-ci1051-20231021-2c709da have issue
3. NootedRed-ci1049-20231021-18739bc no issue

If need, I can provide more log to check if any logic problems (when control the backlight error method?)

**Comment by VisualEhrmanntraut** [comment]
Please try again with latest version.

**Comment by laris** [comment]
testing `NootedRed-ci1055-20231111-dfd2ffa-DEBUG`

clean log and restart system, generated some `gpuRestart` log for your ref:

[gpuRestart_Kernel_2023-11-12_clean-restart-before-22h34m.zip](https://github.com/ChefKissInc/NootedRed/files/13328305/gpuRestart_Kernel_2023-11-12_clean-restart-before-22h34m.zip)

**Comment by laris** [comment]
```
                ,xNMM.          --------------------
               .OMMMMo           OS: macOS 14.1.1 23B81 x86_64
               OMMM0,            Host: Hackintosh (SMBIOS: MacBookPro16,3)
     .;loddo:' loolloddol;.      Kernel: 23.1.0
   cKMMMMMMMMMMNWMMMMMMMMMM0:    Uptime: 7 mins
 .KMMMMMMMMMMMMMMMMMMMMMMMWd.    Packages: 49 (brew)
 XMMMMMMMMMMMMMMMMMMMMMMMX.      Shell: fish 3.6.1
;MMMMMMMMMMMMMMMMMMMMMMMM:       Resolution: 1440x900
:MMMMMMMMMMMMMMMMMMMMMMMM:       DE: Aqua
.MMMMMMMMMMMMMMMMMMMMMMMMX.      WM: Quartz Compositor
 kMMMMMMMMMMMMMMMMMMMMMMMMWd.    WM Theme: Blue (Dark)
 .XMMMMMMMMMMMMMMMMMMMMMMMMMMk   Terminal: iTerm2
  .XMMMMMMMMMMMMMMMMMMMMMMMMK.   Terminal Font: MonacoNFM 14
    kMMMMMMMMMMMMMMMMMMMMMMd     CPU: AMD Ryzen 7 5800U with Radeon Graphics
     ;KMMMMMMMWXXWMMMMMMMk.      GPU: AMD Radeon Graphics
       .cooc,.    .,coo:.        Memory: 8606MiB / 16384MiB




```

Opencore 0.9.6

**Comment by laris** [comment]
upload video to show the behavior about black screen before login.

https://github.com/ChefKissInc/NootedRed/assets/951666/7ccd56bb-0582-4047-87b5-86d2cf259fdd

**Comment by VisualEhrmanntraut** [comment]
Don't see how firmware compression would cause this

**Comment by laris** [comment]
Any troubleshooting and log can track such issue?

**Comment by laris** [comment]
I don't know how to confirm the root cause even Robert help me to build the EFI from scratch.
I think I can bear the blackscreen and seconds to wait to login page come out.
let's close this issue.

**Comment by VisualEhrmanntraut** [comment, marked as answer]
Have you tried using Release build instead of Debug? @laris

**Comment by laris** [comment]
> Have you tried using Release build instead of Debug? @laris

You are right.
I download the release kext and found no issue now.
Should my fault to use debug version
Niubility!

`NootedRed-ci1062-20231203-f39b450-RELEASE.kext`
